UniqueList/UniqueList/List.cs

```diff
@@ -0,0 +1,159 @@
+﻿using UniqueList.Exceptions.DoesntExistException;
+
+namespace Lists;
+
+/// <summary>
+/// Class that represents List data structure.
+/// </summary>
+public class List
+{
+    /// <summary>
+    /// Head of the list.
+    /// </summary>
+    protected Node? head;
+
+    /// <summary>
+    /// Class that represents separate element of the list - node.
+    /// </summary>
+    protected class Node
+    {
+        private int _value;
+        private Node? _next;
+        /// <summary>
+        /// Value of the node.
+        /// </summary>
+
+        public int Value
+        {
+            get { return _value; }
+            set { _value = value; }
+        }
```

Comment:
```suggestion
        public int Value { get; set; }
```
Тогда _value будет не нужно. А ещё кажется, что Node мог бы быть record-ом и писаться в одну строчку.